mindmeld/auto_annotator.py

```diff
@@ -890,6 +890,32 @@ def parse(self, sentence, entity_types, domain: str, intent: str, **kwargs):
             for entity in entities
         ]
 
+    def convert_text_queries_to_processed(self, text_queries):
+        """Converts text queries into processed queries.
+
+        Args:
+            text_queries (list): List of raw text queries.
+        Returns:
+            processed_queries (list): List of processed queries.
+        """
+        processed_queries = []
+        for text_query in text_queries:
+            processed_output = self.nlp.process(text_query)
```

Comment:
I think you can just do this:
```
        query = self.create_query(
            query_text,
            language=self.language,
            locale=locale,
            time_zone=time_zone,
            timestamp=timestamp,
        )
        return self.process_query(
            query, allowed_nlp_classes, dynamic_resource, verbose
        )
```

Reply:
Thanks this is perfect! With this we're able to reduce the function down to just:

```
        return [
            self.nlp.process_query(
                query=self.nlp.create_query(q)
            )
            for q in text_queries
        ]
```